packages/frontend/src/app/App.js

```diff
@@ -495,20 +471,23 @@ class Routing extends Component {
                                 path="/recover-with-link/:accountId/:seedPhrase"
                                 component={ImportAccountWithLinkWrapper}
                             />
+                            <Route
+                                exact
+                                path="/private-key"
```

Comment:
I wonder if we should maybe follow the other route patterns, such as `/recover-private-key` ?

Reply:
Changed